tgui/packages/tgui/interfaces/CrewConsole.jsx

```diff
@@ -86,25 +87,119 @@ export const CrewConsole = () => {
 
 const CrewTable = (props) => {
   const { act, data } = useBackend();
-  const sensors = sortBy(data.sensors ?? [], (s) => s.ijob);
+  const sensors = data.sensors ?? [];
```

Comment:
```suggestion
  const { sensors = [] } = data;
```
prefer destructuring